apps/web/app/components/markdown/markdown.tsx

```diff
@@ -0,0 +1,17 @@
+import ReactMarkdown from 'react-markdown';
+
+import rehypeRaw from 'rehype-raw';
+import remarkGfm from 'remark-gfm';
+import { markdownComponents } from '~/components/markdown/markdown-components';
+
+export default function CustomMarkdown({ children }: { children: string | null | undefined }) {
```

Comment:
```suggestion
export function CustomMarkdown({ children }: { children: string | null | undefined }) {
```

named exports by contention